Dockerfile

```diff
@@ -31,6 +23,10 @@ ADD . /app/source
 WORKDIR /app/source
 RUN bin/build
 
+# remove unnecessary packages & tidy up
+RUN apk del nodejs git wget python make g++
+RUN rm -rf /root/.lein /root/.m2 /root/.node-gyp /root/.npm /tmp/* /var/cache/apk/* /app/source/node_modules
```

Comment:
We can drop the cleanup of `/root/` dirs if we feel the need. Tossed that in to lighten the load on folks running on head; if there are any.